index.html

```diff
@@ -37,154 +36,213 @@ <h1> Hey there. I’m Alex.<span class="green" > <br> I’m a software developer
-           <h4 class="lang-text">Languages </h4>
-               <ul class="lang-text-list">
-                   <li  class="li-lang">Javascript</li>
-                   <li class="li-lang">Ruby</li>
-                   <li class="li-lang">html</li>
-                   <li class="li-lang">css</li>
-               </ul>
+                <div class="about">
+                </div>
+                <div class="about-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+                </div>
+                <div class="butt "><button class="btn btn-resume" type="button"> Get My Resume</button></div>
+            </div>
+                </div>  
+                
+                <div class="skills-container">
+                <article class="project2">
+                <div class="language-icon"><img src="./images/icon-languages (1).png" alt=""></div>
+                <div class="lang-text">Languages </div>
+                    <ul>
+                    <div class="lang-text-list">
+                        <li  class="li-lang">Javascript</li>
+                        <li class="li-lang">Ruby</li>
+                        <li class="li-lang">html</li>
+                        <li class="li-lang">css</li>
+                    </div>
+                    </ul>
+
+                </article>
+                <article class="project2">
+                    <div class="language-icon"> <img src="./images/icon-frameworks (1).png" alt=""></div>
+                    <div class="lang-text">Frameworks </div>
+                    <div class="">
+                    <ul>
+                    <div class="lang-text-list">
+                        <li class="li-lang">Bootstrap</li>
+                        <li class="li-lang">Ruby on rails</li>
+                        <li class="li-lang">Rspec</li>
+                        <li class="li-lang">css</li>
+                        <li class="li-lang">Capybara</li>
+                        <li class="li-lang">Selenium</li>
+                    </div>
+                    </ul>
+                    </div>
+    
+                </article>
+                <article class="project2">
+                    <div class="language-icon"><img src="./images/icon-skills (1).png" alt=""></div>
+                    <div class="lang-text">Skills</div>
+                    <div class="">
+                    <ul>
+                    <div class="lang-text-list">
+                        <li class="li-lang" >CodeKit</li>
+                        <li class="li-lang">Github</li>
+                        <li class="li-lang">Codepen</li>
+                        <li class="li-lang">Gitlab</li>
+                        <li class="li-lang">Terminal</li>
+                    </div>
+                    </ul>
+                    </div>
+    
```

Comment:
[OPTIONAL] Kindly check the responsiveness of your code and make sure it is responsive across the breakpoints 👇
<img width="844" alt="image" src="https://user-images.githubusercontent.com/73854999/206781933-f2d64f70-ab0b-4e33-9409-999d94af9641.png">